apps/web/src/components/Layout/SitewideBanner.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-duplicates */
+import React from "react";
+import isAfter from "date-fns/isAfter";
+import isBefore from "date-fns/isBefore";
+import { useQuery } from "urql";
+
+import { graphql } from "@gc-digital-talent/graphql";
+import { parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { Alert } from "@gc-digital-talent/ui";
+import { useLocale } from "@gc-digital-talent/i18n";
+import { RichTextRenderer, htmlToRichTextJSON } from "@gc-digital-talent/forms";
+
+const SitewideBanner_Query = graphql(/* GraphQL */ `
+  query SitewideBanner {
+    sitewideAnnouncement {
+      isEnabled
+      publishDate
+      expiryDate
+      title {
+        en
+        fr
+      }
+      message {
+        en
+        fr
+      }
+    }
+  }
+`);
+
+const SitewideBanner = () => {
+  const { locale } = useLocale();
+
+  const [{ data }] = useQuery({
+    query: SitewideBanner_Query,
+  });
+
+  // Outage time range, date, starttime
+  const announcementPublishDate = data?.sitewideAnnouncement?.publishDate
+    ? parseDateTimeUtc(data.sitewideAnnouncement.publishDate)
+    : null;
+  const announcementPublishDateIsValid =
+    announcementPublishDate && !Number.isNaN(announcementPublishDate.getTime());
+
+  const announcementExpiryDate = data?.sitewideAnnouncement?.expiryDate
+    ? parseDateTimeUtc(data.sitewideAnnouncement.expiryDate)
+    : null;
+  const announcementExpiryDateIsValid =
+    announcementExpiryDate && !Number.isNaN(announcementExpiryDate.getTime());
+
+  const showMaintenanceBanner =
+    announcementPublishDateIsValid &&
+    isAfter(Date.now(), announcementPublishDate) &&
+    announcementExpiryDateIsValid &&
+    isBefore(Date.now(), announcementExpiryDate);
```

Comment:
Looks like we are not also checking `isEnabled` here so it shows up regardless of that fields value.

Reply:
Lol, minor suggestion.  :grimacing: 
[check isEnabled](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9387/commits/13bffb1f510b01db70267d8493d2db19426295bc)